docs/docs/cloud/cloud-cli.mdx

```diff
@@ -48,6 +48,93 @@ meltano cloud config env delete TAP_GITHUB_AUTH_TOKEN
 
 See the [reserved variables](/cloud/platform#reserved-variables) docs for more details on variables that are reserved for use by Meltano Cloud.
 
+### `notification`
```

Comment:
Thoughts on renaming the command to `notifications`, i.e. make it plural?

Reply:
There was a bit discussion about this a long time ago where we decided that we wanted to go with plural by default, but then had a bunch of exceptional reasons for making particular commands singular. Now it's a mix :sweat_smile:

I'm fine either way, and don't want to hold up this PR.